api/v1alpha1/oadp_types.go

```diff
@@ -184,7 +186,9 @@ type DataProtectionApplicationSpec struct {
 	// BackupImages is used to specify whether you want to deploy a registry for enabling backup and restore of images
 	// +optional
 	BackupImages *bool `json:"backupImages,omitempty"`
-
+	// EnableDataMover is used to specify whether you want to deploy the volume snapshot mover controller and a modified csi datamover plugin
+	// +optional
+	EnableDataMover bool `json:"enableDataMover,omitempty"`
```

Comment:
I was wondering, is it worth adding a new type `TechPreviewConfiguration` that hides this behind a pointer? This would allow us to change this API as we go to GA if upstream velero makes this totally native.

Then if we have other tech preview features in the future we can always use this field.

Reply:
yeah but this will be a supported field, just maybe something we don't keep around in future versions. So I don't think `unsupportedOverrides` is the correct place for it

Reply:
@dymurray I am not sure if `TechPreviewConfigutarion` is the right naming here, I am unsure. Can we not rely on the docs to let the users know that this is a tech-preview feature ? Or is it the right way to do from the API side as you suggested ?

Reply:
yeah but this will be a supported field, just maybe something we don't keep around in future versions. So I don't think unsupportedOverrides is the correct place for it

Reply:
Cool, I like `features` more and its combination with the docs. Will be similar to what `Velero` does

Reply:
@dymurray updated the PR, PTAL, Thanks !